modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,460 @@
+= Monitoring Metrics from Anypoint Insights
+:page-notice-banner-message: For Japan Cloud and Canada Cloud instances of Anypoint Platform, Anypoint Insights is available in the Starter package. For US and EU clouds, Anypoint Insights remains a Beta release available in the Advanced and Titanium tiers at this time.
+:selector-all-dashboards: This selector is available to all Insights dashboards.
+:download-chart-data: You can download most chart data to CSV or XSL.
+
+Monitor performance data and issues across your deployed Mule apps and API implementations (_entities_) with Anypoint Insights, in Anypoint Monitoring: 
+
+* Check metrics across multiple Mule apps and API implementations
++
+Get up to the last 7 days (Advanced package) or 24 hours (Starter package) of high-level metrics on requests, errors, and responses. Use filters to narrow your results. {download-chart-data}
+* Check metrics on individual Mule apps and API implementations
++
+Get up to the last 30 days of metrics on latency, requests, responses, errors, Mule messages and the JVM for Mule apps, and for API implementations, data on policy violations, performance, and client apps. {download-chart-data}
+
+[[entities]]
+include::partial$include-entities.adoc[tags="entity-definition"]
+
+[[dashboard-overview]]
+//don't change order of numbers in graphic or calloutlist
+image::insights.png["Anypoint Insights UI"]
+
+[calloutlist]
+. Dashboard that provides an overview of metrics across of your deployed Mule apps and APIs
+. Environment selector for your deployments (such as Sandbox or Production)
+. <<filter, Filters>> to narrow the data in charts
+. Performance <<charts-performance, metrics and trends>> across your deployed Mule apps and APIs
+. <<entities-list, List of each Mule app and API>> that you can click to display <<charts-detailed, detailed metrics>> on a specific deployment
+. Selector for the time range covered by the charts 
++
+{selector-all-dashboards}
+. Selector for auto-refresh interval, displayed when auto-refresh is enabled 
++
+{selector-all-dashboards}
+
+[[prereqs]]
+== Before You Begin
+
+Required prerequisites:
+
+* An Anypoint Platform account in a region that supports Anypoint Insights
+
+** {empty}
+include::partial$include-entities.adoc[tags="anypoint-regions-hyperforce"] 
+
+** {empty}
+include::partial$include-entities.adoc[tags="anypoint-regions-useu"]
+
+* A Mule app or API running on a supported version of Mule Runtime engine (see xref:release-notes::mule-runtime/mule-esb.adoc[]) and deployed to a supported target:
```

Comment:
Question @sathishpaulleo: Is the API is a Mule Gateway API, or can it be another type, like Flex Gateway?

Reply:
from Sathish: can be either (just these)